README.md

```diff
@@ -49,12 +49,45 @@ Add this to your `<project>/testem.js`...
   },
 ```
 
+## Configuration
+
+By default, `ember-backstop` will _not_ take screenshots. To enable taking screenshots do the following:
```

Comment:
Let's please not introduce breaking behavior here.   We can allow each implementation to "opt-in" by returning `diableBackstop = true` in `enviornment.js`